jobs/cloud_controller_ng/spec

```diff
@@ -1124,3 +1124,7 @@ properties:
   copilot.temporary_istio_domains:
     description: "Array of domains (strings) to be used for routes propagated to the Envoy/Istio routing tier. These routes will not be propagated to Gorouter."
     default: []
+
+  cc.experimental.thread_count:
+    description: "How many threads a single cloud controller instance will attempt to use. Alter at your own peril."
```

Comment:
Can you make it more clear that these threads are for the Thin server's thread pool?
https://github.com/cloudfoundry/cloud_controller_ng/pull/1356/files#diff-fe19359efebf8a93120e2e6a23eae813R163

We use Ruby threads in other places like our [Workpools](https://github.com/cloudfoundry/cloud_controller_ng/blob/master/lib/utils/workpool.rb) and this has no power there.

I think `cc.experimental.thin_server_thread_pool_size` might be more clear of a property name.